contracts/programs/bubblegum/src/state/authority.rs

```diff
@@ -0,0 +1,88 @@
+use crate::error::BubblegumError;
+use anchor_lang::prelude::*;
+
+pub const GUMMYROLL_TREE_AUTHORITY_SIZE: usize = 304 + 8;
+pub const APPEND_ALLOWLIST_SIZE: usize = 5;
```

Comment:
I think we need to up this number

Reply:
Honestly this scheme doesn't even make sense

Reply:
yea we can also do it with the PDA system (need more accounts)